markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[13:47](https://www.youtube.com/watch?v=kBoC6QBU-M0t=827)
+I think that's everything I really wanted to show you on this very basic example. I'm going to quickly go to another example that I think is quite helpful to explore, which is actually on the website. If I go to the website repository we can also run it from here. I'll just show a few more features that I think are really interesting to see. We can actually live render the whole website and show that the whole website is changing and we can change the underlying markdown. This is quite helpful when we've been developing the website so hopefully this will quickly load and I can show you this example.
+
+[14:31](https://www.youtube.com/watch?v=kBoC6QBU-M0t=871)
+I will show you in this repository. We see the gitpod.yml file, this is what I showed you before. There's many things you can put in these yml files and I think the best way to learn about this is to go to the gitpod.io website or by looking at other people's gitpod.yml files. There you can see exactly what people are doing.
+
+[14:56](https://www.youtube.com/watch?v=kBoC6QBU-M0t=896)
+Before I showed you image. Image was pulling a docker container, but you also can initiate tasks. You can do lots of different commands related to docker. You can open ports to expose ports so that we can look at web browser and share a particular web browser html rendering. The other thing are the VSCode extensions. These extensions are all over here and you can make sure that each repository that you make into a Gitpod environment has all the different code linting services, adding different particular pieces of code that are very useful to have each time. You can add these extensions manually in this gitpod.yml.
+
+[15:43](https://www.youtube.com/watch?v=kBoC6QBU-M0t=943)
+You can also set all the github settings that you would need to change. Just very quickly, because I know I'm probably running over time, here in the remote browser we can set this to being a live preview. Okay, no you can't... Probably I have done something wrong. There we go! Within this repository we're now making a live view of the actual repository that's changed and then if I wanted to go through the code into the markdown I can actually change the markdown. Like I showed you before you can actually change what's happening in this live preview of the nf-core website. I think this is super useful as well.
```

Comment:
```suggestion
You can also set all the GitHub settings that you would need to change. Just very quickly, because I know I'm probably running overtime, here in the remote browser we can set this to being a live preview. Okay, no you can't... Probably I have done something wrong. There we go! Within this repository, we're now making a live view of the actual repository that's changed and then if I wanted to go through the code into the markdown I can actually change the markdown. Like I showed you before you can actually change what's happening in this live preview of the nf-core website. I think this is super useful as well.
```